tools/circle_plus_gen/schema/__init__.py

```diff
@@ -1,5 +1,5 @@
 # In this directory, *_generated.py is auto generated by flatc.
 #
-# * flatc version 23.5.26 is used
-# ./flatc --python --gen-onefile --gen-object-api ../../../nnpackage/schema/circle_schema.fbs
-# ./flatc --python --gen-onefile --gen-object-api ../../../runtime/libs/circle-schema/include/circle_traininfo.fbs
+# (For devleopers) You could easily update *_generated.py using '_gen.py' in this folder.
+#   $ pip3 install wget
+#   $ python3 _gen.py
```

Comment:
The updates in this PR (https://github.com/Samsung/ONE/pull/13131) are made by this script(`_gen.py`).